<launch>
    <arg name="mission_config" default="default"/>
    <arg name="load_map" default="false"/>

    <node name="graphslam" pkg="graphslam" type="graphslam" output="screen">
        <param name="load_map" type="bool" value="$(arg load_map)"/>
        <rosparam command="load" file="$(find graphslam)/config/graphslam_$(arg mission_config).yaml"/>
        <param name="map_path" type="str" value="$(find graphslam)/maps"/>
    </node>
    
    <!-- <include file="$(find svm)/launch/svm_python.launch" unless="$(arg load_map)"/> -->
    
    <group if="$(arg load_map)">
        <rosparam command="load" file="$(find graphslam)/maps/$(arg mission_config).yaml"/> 
    </group>
</launch>